services/site/src/pages/index.tsx

```diff
@@ -156,6 +157,28 @@ const Home: React.VoidFunctionComponent<HomeProps> = ({ fmType }) => {
                 challenges={hardChallenges}
               />
             )} */}
+
+            <p className="mb-8 md:mb-0">
+              <span className="sr-only">Disclaimer:</span>
+              <span aria-hidden="true" className="text-xl text-grey">
+                *
+              </span>{" "}
+              Sorry, at the moment, we do not have more advanced challenges. Our small team is dedicated to producing more content for you. We will
+              announce new challenges on our{" "}
+              <Link href={"https://twitter.com/a11yphant"}>
+                <a
+                  className={clsx(
+                    "text-light font-sans font-normal border-light",
+                    "transition-colors duration-300",
+                    "hover:text-primary-light hover:border-transparent",
+                    "focus-rounded-instead-of-underline",
+                  )}
+                >
+                  Twitter account
+                </a>
+              </Link>
+              . Can't wait or know anyone that could help us? Slide into our DMs.
+            </p>
```

Comment:
```suggestion
            <p className="mb-8 md:mb-0">
              <span className="sr-only">Disclaimer:</span>
              <span aria-hidden="true" className="text-xl text-grey">
                *
              </span>{" "}
              Currently, we only offer easy challenges. However, our small team is dedicated to producing more content for you. We will announce new
              challenges on our{" "}
              <Link href={"https://twitter.com/a11yphant"}>
                <a
                  className={clsx(
                    "text-light font-sans font-normal border-light",
                    "transition-colors duration-300",
                    "hover:text-primary-light hover:border-transparent",
                    "focus-rounded-instead-of-underline",
                  )}
                >
                  Twitter account
                </a>
              </Link>
              . <br />
              Want to help us? Slide into our DMs.
            </p>
```

An alternative suggestion.
- The structure of the sentence seems strange. Also, it sounds a bit apologetic starting with "sorry". I'd prefer a more plain, factual phrasing.
- I had to read the question in the end twice. It's a bit hard to understand, especially with the line break on the page (at least on my screen). I think we should keep it short and simple
- I'd prefer a line break before the question